src/main/scala/io/iohk/ethereum/domain/SignedTransaction.scala

```diff
@@ -16,43 +18,67 @@ object SignedTransaction {
   val AddressLength = 20
   val LastByteOfAddress: Int = FirstByteOfAddress + AddressLength
   val negativePointSign = 27
+  val newNegativePointSign = 35
   val positivePointSign = 28
-  val allowedSignValues: Set[Int] = Set(negativePointSign, positivePointSign)
-
+  val newPositivePointSign = 36
 }
 
 case class SignedTransaction(
   tx: Transaction,
-  pointSign: Byte, //v - 27 or 28 according to yellow paper, but it is 37 and 38 in ETH
+  pointSign: Byte, //v
   signatureRandom: ByteString, //r
   signature: ByteString /*s*/) {
 
   import SignedTransaction._
 
-  lazy val bytesToSign: Array[Byte] =
+  lazy val bytesToSign: Array[Byte] = if (pointSign == negativePointSign || pointSign == positivePointSign) {
+    //global transaction
+    crypto.sha3(
+      rlpEncode(RLPList(
+        tx.nonce,
+        tx.gasPrice,
+        tx.gasLimit,
+        tx.receivingAddress.toArray,
+        tx.value,
+        tx.payload)))
+  } else {
+    //chain specific transaction
     crypto.sha3(
       rlpEncode(RLPList(
-                  tx.nonce,
-                  tx.gasPrice,
-                  tx.gasLimit,
-                  tx.receivingAddress.toArray,
-                  tx.value,
-                  tx.payload)))
+        tx.nonce,
+        tx.gasPrice,
+        tx.gasLimit,
+        tx.receivingAddress.toArray,
+        tx.value,
+        tx.payload,
+        Config.Blockchain.chainId,
+        0,
+        0)))
+  }
 
 
   lazy val recoveredAddress: Option[Array[Byte]] =
     recoveredPublicKey.map(key => crypto.sha3(key).slice(FirstByteOfAddress, LastByteOfAddress))
 
-  //todo implement EC-72
-  lazy val recoveredPublicKey: Option[Array[Byte]] = if (allowedSignValues.contains(pointSign)) {
+  /**
+    * formula for calculating pointt sing new way
+    * v = CHAIN_ID * 2 + 35 or v = CHAIN_ID * 2 + 36
+    */
+  lazy val recoveredPointSign: Option[Int] = pointSign match {
```

Comment:
Could save few key strokes with if/else here. We don't leverage 'match', might even be clearer, although it's clear right now.